tests/reference/js_defer_onload.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <!-- Ensure that onloads in html tags are executed without any errors. -->
+  </head>
+  <body>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.js"></script>
+    <script type="text/javascript" >
+      window.onload = function () {
+        document.writeln('onload');
+      }
+    </script>
+    <script type="text/javascript" >
+      var node = document.createElement("script");
+      node.async = true;
+      node.src = "https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.js";
+
+      $(document).find("body").append(node); // it fails, the script is not inserted but fetch in Ajax !!!
```

Comment:
When I replace the `document.writeln` by an other instruction like `$('body').append('onload');` everything is working okay, I have my message and the script is correctly inserted.

![image](https://user-images.githubusercontent.com/1662759/30582062-a85b1386-9d22-11e7-8525-307be80809a9.png)

So I think that the `writeln` is executed after the insertion of the tag so it will remove every thing and this is what you have observed by your comment ` it fails, the script is not inserted but fetched in ajax`